### ISSUE TITLE
Electron app generation error. Node 12.

### ISSUE THREAD

**Issue by ph55** [nstudio/xplat#141]
When generating electron app, error thrown:

```
> fs-xattr@0.1.17 install /Users/sasha/Dev/apps-client/node_modules/fs-xattr
> node-gyp rebuild

  CXX(target) Release/obj.target/xattr/src/async.o
../src/async.cc:35:15: warning: 'Call' is deprecated [-Wdeprecated-declarations]
    callback->Call(1, argv);
              ^
../../nan/nan.h:1739:3: note: 'Call' has been explicitly marked deprecated here
  NAN_DEPRECATED inline v8::Local<v8::Value>
  ^
../../nan/nan.h:104:40: note: expanded from macro 'NAN_DEPRECATED'
# define NAN_DEPRECATED __attribute__((deprecated))
                                       ^
../src/async.cc:52:46: error: no matching member function for call to 'ToObject'
    v8::Local<v8::Object> bufferObj = value->ToObject();
                                      ~~~~~~~^~~~~~~~
/Users/sasha/Library/Caches/node-gyp/12.6.0/include/node/v8.h:2532:44: note: candidate function not viable: requires single argument 'context', but no arguments were
      provided
  V8_WARN_UNUSED_RESULT MaybeLocal<Object> ToObject(
                                           ^
/Users/sasha/Library/Caches/node-gyp/12.6.0/include/node/v8.h:2546:31: note: candidate function not viable: requires single argument 'isolate', but no arguments were
      provided
                Local<Object> ToObject(Isolate* isolate) const);
                              ^
../src/async.cc:129:15: warning: 'Call' is deprecated [-Wdeprecated-declarations]
    callback->Call(2, argv);
              ^
../../nan/nan.h:1739:3: note: 'Call' has been explicitly marked deprecated here
  NAN_DEPRECATED inline v8::Local<v8::Value>
  ^
../../nan/nan.h:104:40: note: expanded from macro 'NAN_DEPRECATED'
# define NAN_DEPRECATED __attribute__((deprecated))
                                       ^
../src/async.cc:186:15: warning: 'Call' is deprecated [-Wdeprecated-declarations]
    callback->Call(2, argv);
              ^
../../nan/nan.h:1739:3: note: 'Call' has been explicitly marked deprecated here
  NAN_DEPRECATED inline v8::Local<v8::Value>
  ^
../../nan/nan.h:104:40: note: expanded from macro 'NAN_DEPRECATED'
# define NAN_DEPRECATED __attribute__((deprecated))
                                       ^
3 warnings and 1 error generated.
make: *** [Release/obj.target/xattr/src/async.o] Error 1
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:196:23)
gyp ERR! stack     at ChildProcess.emit (events.js:203:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:272:12)
gyp ERR! System Darwin 18.6.0
gyp ERR! command "/usr/local/Cellar/node/12.6.0/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/sasha/Dev/apps-client/node_modules/fs-xattr
gyp ERR! node -v v12.6.0
gyp ERR! node-gyp -v v5.0.3
gyp ERR! not ok
```

Probably related to https://github.com/LinusU/node-appdmg/issues/176,

**Comment by NathanWalker** [comment]
@ph55 can you provide exact command you used to generate electron app which lead to this error?  I've been thinking of bumping all electron versions as I've noticed that `appdmg` 0.6.0 is in the latest `electron-installer-dmg` which should be only package which relies on that package but I believe if that's the case you would only run into issue if you tried to *build* a mac installer version. However you mentioned just generating an electron app?

**Comment by ph55** [comment]
@NathanWalker sure. Running the regular `ng g app name`. Then picking `electron` from options. Then specifying web app name. 

![image](https://user-images.githubusercontent.com/1386622/62418461-b40fba00-b671-11e9-9dd1-c61328a573c0.png)
![image](https://user-images.githubusercontent.com/1386622/62418469-cb4ea780-b671-11e9-9007-0125a51089c1.png)

Ye, I think, the `electron-installer-dmg@2.0.0` relies on low version of `appdmg` which relies on low version of `fs-xattr` which throws error 🙂

**Comment by NathanWalker** [comment]
Fixed in `8.0.4` now. If you reinstall latest schematics in your workspace the above issue should be fixed and any other issues you may have been having with electron + angular (including the xplat layer generation) 👍
